### PR TITLE
Added 'min-width' for easy image widget

### PR DESCRIPTION
**Pull request by f1ames**
## What is the purpose of this pull request?

Bug fix

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_contributing_code-section-tests),
[how to set the testing environment](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests) and
[how to create tests](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests-section-creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [x] Manual tests

## What changes did you make?

Added 'min-width' for easy image widget so it does not scale to very tiny sizes when placed in smaller containers like table cells.

Closes: #1553

### CODE REVIEW

File: tests/plugins/easyimage/manual/sideimagebig.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<h2>Classic editor</h2>
```
Comment (mlewand) [inline tests/plugins/easyimage/manual/sideimagebig.html]: This test is pretty much broken on IE11. Table resize does not work here,

File: tests/plugins/easyimage/manual/sideimagebig.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@bender-tags: 4.9.0, feature, 932
```
Comment (mlewand) [inline tests/plugins/easyimage/manual/sideimagebig.md]: Both tests are bug tests more than a feature.

File: tests/plugins/easyimage/manual/sideimagebig.html (hunk 1 above)
Comment (f1ames) [reply tests/plugins/easyimage/manual/sideimagebig.html]: Looks like it is not the issue with added styles, but basically an issue with IE11 and easyimage widget inside table. Without `min-width` and even `max-width` table is still not resizable.

Comment (mlewand) [reply tests/plugins/easyimage/manual/sideimagebig.html]: That's an interesting issue, could you extract this? I think that we simply should change manual test concept here to have it solved.

Comment (f1ames) [reply tests/plugins/easyimage/manual/sideimagebig.html]: Change the test to use smaller image so resizing is possible (but image still scales) and add proper info.

**Comment by mlewand** [comment]
I have rebased the branch onto latest `t/932-2961`.

**Comment by f1ames** [comment]
@mlewand I adjusted tests description, now I think it should fit such cases.

**Pull request merged**